src/main/resources/sqlUpdates/DatabaseChangeGuidelines.md

```diff
@@ -39,13 +42,13 @@ There are various naming conventions, please [IntegrationAndAppNotes](integratio
 - If altering a table structure ( add /remove change columns) is there an audit (_AUD) table
  that also needs to be changed in the same way?
 - when adding/modifying audit table columns, remember that they don't generally have not-null constraints (apart from id/REV columns).
--  If you are adding such constraint make sure you know the consequences, and double check if constraint is actually applied on your local database when testing.
+- If you are adding such constraint make sure you know the consequences, and double check if constraint is actually applied on your local database when testing.
 
 ### Have you set a context? Values are:
 - no value. Will run on all databases in all environments. E.g. static  reference data
 - Suitable for adding lookup static data to all test/dev/production databases. E.g., permissions.
   - `run` for general schema changes - adding/removing columns/ tables /keys
-  - `dev-test` change should only be applied during unit test runs - e.g., loading up test data.
+  - `dev-test` change should only be applied on local dev environment, or during unit test runs - e.g., loading up test data.
```

Comment:
So going forward, the vast majority of changes will only have the `run` context as it runs all of the time, as oppose to now where many have both `run,dev-test`? And `dev-test` will only be for the specific case you mention here?

Reply:
yes, going forward we shouldn't need combined `run,dev-test` context at all (i.e. can be replaced with just `run`), and only occasional `dev-test` for change that should be run for unit tests or localhost.